src/test/scala/it/pagopa/interop/partyprocess/ExternalApiSpec.scala

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
I tried to create a test class for the ExternalApi copying from PartyProcess test, but if uncomment this it's tests goes to timeout and all the other tests fail too
But I'm not able to understand why

Could you please help me?
Otherwise let's delete this class

@beetlecrunch @TonioGela @galales

Reply:
I just applied a fix for the tests. 
This part must be analyzed more, but we are in a hurry.